src/components/code-editor/code-editor.tsx

```diff
@@ -168,13 +188,32 @@ export class CodeEditor {
                 name: 'application/json',
                 json: true,
             };
+            if (this.lint) {
```

Comment:
Should we limit linting for use when `language` is `json` since I don't know what happens if we list when language is something else?

Reply:
This does not limit anything but adds the necessary linter when using json as the language.
If we add other linters I guess we add the lint-markers gutter in that case?
I'm not sure if I understand your comment correctly.

I added the lint prop as an opt-in to avoid breaking any current use. To me it seems reasonable to always have linting enabled for json content, but I guess it's better to let the consumer be in control of that behavior.

Note: This PR only add support for json linting

Reply:
What I meant is that if you enable linting it will be enabled regardless of what `this.language` is. And as far as I can understand the linter you add in this PR does only support `json`? So what happens if you combine `language = typescript` with `lint = true`?

Reply:
I see what you mean but nothing will happen. Even though `lint` is `true` in the editor options there will be no support since no lint gutter is added + the codemirror lint addon for the language is not imported. In addition to that, a linter implementation is also needed. All this was added for json but not for other languages. 
If we would import the lint addon for typescript it would also log an error in the console if we don't provide a linter in the window object (required by the lint addon).